mysite/templates/generic_base.html

```diff
@@ -155,7 +155,8 @@
         <p><a href="{% url 'help-api' %}">{% trans "Open data API" %}</a>
             &middot; <a href="{% url 'help-about' %}">{% trans "About" %}</a>
             &middot; <a href="https://github.com/DemocracyClub/yournextrepresentative/issues">{% trans "Issue tracker" %}</a>
-            &middot; <a href="{% url 'help-privacy' %}">{% trans "Privacy policy" %}</a></p>
+            &middot; <a href="{% url 'help-privacy' %}">{% trans "Privacy policy" %}</a>
+            &middot; <a href="{% url 'help-photo-policy' %}">{% trans "Photo policy" %}</a></p>
```

Comment:
I don't this this is imprant enough to include as a separate item in the footer, actually. Being linked to from the About page is fine.